client/src/app/site/pages/meetings/services/active-meeting.subscription.ts

```diff
@@ -25,8 +25,6 @@ export function getActiveMeetingSubscriptionConfig(id: Id, settingsKeys: string[
                 `welcome_title`,
                 `welcome_text`,
                 `enable_anonymous`,
-                { templateField: `logo_$_id` },
-                { templateField: `font_$_id` },
```

Comment:
This should probably hold all the replacement `logo_(...)_id` and `font_(...)_id` fieldnames (see `MEETING_MEDIAFILE_USAGE_ID_KEYS` in `meeting.constants.ts`)